### PR TITLE
MEPTS-708: M&Q Categoria 13 Numerator Part1_2

### PR DESCRIPTION
**Pull request by amosh101**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 1]
```diff
@@ -2980,6 +2980,8 @@ public CohortDefinition getMQ12DEN(Integer den) {
   }
 
   /**
+<<<<<<< HEAD
+=======
    * <b>MQC13Part3B1</b>: Melhoria de Qualidade Category 13 Deniminator B1 <br>
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: Guess you did not mean to put this here

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@amosh101 can you please remove this comments

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java (hunk 1 above)
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: Removed

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**